src/main/java/org/ehcache/integrations/shiro/EhcacheShiroManager.java

```diff
@@ -29,43 +30,96 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.net.MalformedURLException;
 import java.net.URL;
 
+/**
+ * Shiro {@link CacheManager} implementation using the Ehcache 3.0 framework for all cache functionality
```

Comment:
Would use `3.x` here instead of `3.0` especially as prop replacement will end up requiring `3.2`